src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/movements/TemporaryAbsenceApplicationMapping.kt

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.movements
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.Transient
+import org.springframework.data.domain.Persistable
+import java.time.LocalDateTime
+import java.util.*
+
+data class TemporaryAbsenceApplicationMapping(
+
+  @Id
+  val dpsApplicationId: UUID,
```

Comment:
very minor - I tend to always make the DPS key just a String since we don't care what their type is, so works for strings. long, uuid